tests/test_example_dags.py

```diff
@@ -32,6 +32,7 @@ def session():
         "example_snowflake_partial_table_with_append",
         "example_sqlite_load_transform",
         "example_dynamic_map_task",
+        "example_dynamic_task_template",
```

Comment:
@kaxil @tatiana @dimberman  - we should find a way to split these dags and run them in parallel/parallel groups since we will be adding a lot of them going forwards and this can slow CI. WDYT?

Reply:
agreed. Can you create a ticket on that please